Mage.Sets/src/mage/sets/LoveYourLGS2021.java

```diff
@@ -0,0 +1,34 @@
+package mage.sets;
+
+import mage.cards.ExpansionSet;
+import mage.constants.Rarity;
+import mage.constants.SetType;
+
+/**
+ * https://scryfall.com/sets/plg21
+ */
+public class LoveYourLGS2021 extends ExpansionSet {
+
+    private static final LoveYourLGS2021 instance = new LoveYourLGS2021();
+
+    public static LoveYourLGS2021 getInstance() {
+        return instance;
+    }
+
+    private LoveYourLGS2021() {
+        super("Love Your LGS 2021", "PLG21", ExpansionSet.buildDate(2021, 1, 1), SetType.PROMOTIONAL);
```

Comment:
Scryfall seems to think the date is 2021, 6, 22

Reply:
thanks